### DISCUSSION TITLE
when I run quasar dev it stuck and don't show changes

### DISCUSSION THREAD

**Discussion by lucho19jose** [quasarframework/quasar#9871] [CLI - BEX mode]
hi guys, I am using quasar cli and when I watch my app in the browser it run normally, but when I make changes it doesn't show.
it appear:
[webpack-dev-server] App hot update...
and it keep  stuck
![image](https://user-images.githubusercontent.com/46167630/123899628-2e641a00-d92d-11eb-9155-c05e77ea20ef.png)

**Comment by gaguerrero** [comment]
Same problem here, any solution?

**Comment by ontwikkelfabriek** [comment]
@lucho19jose @gaguerrero 

I presume you are using **BEX** mode? 

Did you read this about `HMR`, `BEX` and Chrome/Firefox differences:
https://quasar.dev/quasar-cli/developing-browser-extensions/build-commands#hot-module-reloading-hmr-

**Reply by gaguerrero** [reply]
TThanks I will read it!
